modules/release-notes/single-source-release-notes.py

```diff
@@ -54,7 +54,7 @@
 )
 # Delete old file files.
 fileList = glob.glob(
-  modules_dir + 'snip-*.adoc'
+  modules_dir + 'snip-*-rhidp-*.adoc'
```

Comment:
will this delete more than the CVE files? like known issues and removed functionality?

Reply:
Yes it was deleting the Asciidoc files with the CVE content.